site/en/docs/devtools/performance/reference/index.md

```diff
@@ -96,17 +86,17 @@ functions that were called during the recording. To disable these call stacks:
 2.  Enable the **Disable JavaScript Samples** checkbox.
 3.  Take a recording of the page.
 
-Figure 7 and Figure 8 show the difference between disabling and enabling JavaScript samples. The
+The following screenshots show the difference between disabling and enabling JavaScript samples. The
 **Main** section of the recording is much shorter when sampling is disabled, because it omits all of
 the JavaScript call stacks.
 
-{% Img src="image/admin/ggxgXYwyRlwqZ64RQjYR.png", alt="An example of a recording when JS samples are disabled", width="800", height="626" %}
+{% Img src="image/admin/ggxgXYwyRlwqZ64RQjYR.png", alt="An example of a recording when JS samples are disabled.", width="800", height="626" %}
 
-**Figure 7**. An example of a recording when JS samples are disabled
+The example above shows a recording with disabled JS samples.
 
-{% Img src="image/admin/mDRqRgX1BgCgS3UjWSzm.png", alt="An example of a recording when JS samples are enabled", width="800", height="781" %}
+{% Img src="image/admin/mDRqRgX1BgCgS3UjWSzm.png", alt="An example of a recording when JS samples are enabled.", width="800", height="781" %}
```

Comment:
I would consider putting these into a figure with a figcaption, the style guide suggests not using "above" and "below", but these statements "the example above" seem more like figcaptions, and that would tie them to the image without the directional language.